operations/mimir-mixin/config.libsonnet

```diff
@@ -649,6 +649,8 @@
       'debug_pprof',
     ],
 
+    ingester_read_path_routes_regex: '/cortex.Ingester/Query(Stream)?|/cortex.Ingester/MetricsForLabelMatchers|/cortex.Ingester/LabelValues|/cortex.Ingester/MetricsMetadata',
```

Comment:
I know this is what we use in the Reads dashboard but I think it's missing several stuff (e.g. ActiveSeries, UserStats, ...).

Reply:
Will send separate PR to fix, since it's unrelated to this PR.

Reply:
Fixed in https://github.com/grafana/mimir/pull/7676.